laminas-ci.schema.json

```diff
@@ -328,14 +331,19 @@
             }
         },
         "stablePHP": {
-                "type": "string",
-                "minLength": 1,
-                "title": "The PHP version to be used for stable checks",
-                "description": "This PHP version is used for all QA check jobs. The default depends on the `composer.json` of the project and usually reflects the minimum supported PHP version of that project.",
-                "examples": [
-                    "8.0"
-                ]
-            },
+            "type": "string",
+            "minLength": 1,
+            "title": "The PHP version to be used for stable checks",
+            "description": "This PHP version is used for all QA check jobs. The default depends on the `composer.json` of the project and usually reflects the minimum supported PHP version of that project.",
+            "examples": [
+                "8.0"
+            ]
+        },
+        "backwardCompatibilityCheck": {
+            "type": "boolean",
+            "title": "Flag to enable/disable backwards compatibility check",
+            "description": "This flag enables/disables backwards compatibility check using roave/backward-compatibility-check."
+        },
```

Comment:
Should we declare a default, so it is self-documenting on that?

Reply:
Is that only `"default": false`?
If so, ofc we can 👍🏼